src/screening/IdentifierFilters.jsx

```diff
@@ -146,10 +155,12 @@ class ScreeningFilters extends React.Component {
               list={PATIENT_IDENTIFIERS_SUFFIX} 
               placeholder=" "
             />
+            {searchType !== 'server' &&
             <button
```

Comment:
@rhenshaw56 I haven't looked at the code fully and/or tested, but I thought we were going to want to show the button if it *is* a server search (ie as on the Search Page)

Reply:
yeah, we're still doing that but this is for the filters and not for the whole form